main.py

```diff
@@ -18,18 +18,19 @@
 
 
 def main(WorldClass):
-    script_path = Path(__file__).resolve().parent
-    state_pattern = script_path / 'state' / 'last_{filename}'
 
     display = Display(TITLE, WORLD_SIZE, SCALE)
     clock = pygame.time.Clock()
 
     world = WorldClass(WORLD_SIZE, display)
 
+    episode_count = 0
     while True:
```

Comment:
`for episode_count in itertools.count():` is een mooi alternatief

Reply:
Ja, is mooi.